modules/auth/saas/settings/settings.tsx

```diff
@@ -27,9 +31,12 @@ const Settings = () => {
     resolver: yupResolver(settings_schema(t)),
   });
 
-  const [showPassword, setShowPassword] = useState(false);
-  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
-
+  const handleRevokeOAuth = async (): Promise<void> => {
+    const response = await Service.require("oauth").revokeOauth();
+    if (response) {
+      await Service.require("oauth").logout();
+    }
```

Comment:
There is no successfull message or anything.

If I revoke my google oauth, I may want to register it again dont I? 

Nothing happens in the page seems is broken